tests/pytorch/test_nn.py

```diff
@@ -1621,4 +1621,27 @@ def test_dgn_conv(in_size, out_size, aggregators, scalers, delta,
     aggregators_non_eig = [aggr for aggr in aggregators if not aggr.startswith('dir')]
     model = nn.DGNConv(in_size, out_size, aggregators_non_eig, scalers, delta, dropout,
         num_towers, edge_feat_size, residual).to(dev)
-    model(g, h, edge_feat=e)
\ No newline at end of file
+    model(g, h, edge_feat=e)
+
+
+@parametrize_idtype
+@pytest.mark.parametrize('meta_path', [['uc','cu'],['uc','cp','pc','cu'],['cp','pc','cp','pc']])
+@pytest.mark.parametrize('window_size', [1,3])
+@pytest.mark.parametrize('negative_samples', [1,3])
+@pytest.mark.parametrize('min_count', [0,1])
+@pytest.mark.parametrize('emb_dim', [1,  100])
+@pytest.mark.parametrize('node_repeat', [1,  3])
+def test_Metapath2vec(emb_dim,meta_path,window_size,negative_samples,min_count,node_repeat,idtype):
```

Comment:
It may not be necessary to test different options for all these arguments. This yields a large product space.